### DISCUSSION TITLE
454

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#176] [Comments]
# 454

문제 쿠키에 JWT 를 저장하고, 토큰값 기반 로그인을 구현 해 두었다. 그런데 Chrome 이나 Firefox, Opera 등 다른 브라우저에서는 다 문제 없이 작동하는데 유독 사파리에서만 동작이 안된다. Network를 확인 해 보면, 로그인 성공시 정상적으로 Set-Cookie가 내려 온다. 그런데 저장된 쿠키를 확인 해보면 방금 저장하도록 한 쿠키값이 저장이 되어 있지 않다. 한가지 특이한건, localhost가 아닌 운영중인 서버에서는 Safari 에서도 쿠키가 정상적으로 저장되었다는 것이다. 원인 일단 용의자는 아래와 같이 잡고 몇가지 테스트를 진행해 보았다. domain SameSite Secure 제일 먼저 Domain을 확인 해 봤는데, 기존에는 위 스샷처럼 Response Cookie에..

https://shanepark.tistory.com/454

**Comment by ranunb** [comment]
안녕하세요. 
덕분에 문제를 잘 해결했습니다. 너무 감사합니다. !
